common-rest/src/main/java/org/uniprot/api/rest/service/query/UniProtQueryProcessor.java

```diff
@@ -43,7 +42,8 @@
 public class UniProtQueryProcessor implements QueryProcessor {
     public static final String IMPOSSIBLE_FIELD = "NOT_REAL_FIELD";
     public static final String UNIPROTKB_ACCESSION_FIELD = "accession";
-    private static final Pattern CLEAN_QUERY_REGEX = Pattern.compile("(?:^\\()|(?:\\)$)");
+    private static final Pattern CLEAN_QUERY_REGEX =
```

Comment:
I think you can remove this CLEAN_QUERY_REGEX constant from here, because it is not used, right?

Reply:
removed